src/app/teacher/tags-filter/tags-filter.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatDividerModule } from '@angular/material/divider';
+import { SharedModule } from '../../modules/shared/shared.module';
+import { Tag } from '../../domain/tag';
+import { AbstractTagsMenuComponent } from '../abstract-tags-menu/abstract-tags-menu.component';
+
+@Component({
+  selector: 'tags-filter',
+  templateUrl: './tags-filter.component.html',
+  styleUrls: ['./tags-filter.component.scss'],
+  standalone: true,
+  imports: [CommonModule, MatDividerModule, SharedModule]
```

Comment:
This component imports SharedModule, but it only needs some of the components and modules within it. Is it possible to only import those dependencies instead?

Reply:
Right now I don't think you can because we need SearchBarComponent which is not standalone and is declared in SharedModule.

Reply:
Ok- let's make SearchBarComponent into a standalone component soon.